### ISSUE TITLE
Crashes if posting with x-www-form-urlencoded

### ISSUE THREAD

**Issue by volkert** [juice-shop/juice-shop#152]
The server crashes (unfortunately with no log) if posting with `x-www-form-urlencoded`.

See:
<img width="753" alt="screen shot 2016-01-14 at 18 00 31" src="https://cloud.githubusercontent.com/assets/80850/12331260/19b0afd2-bae9-11e5-83c9-e3d87bf1f9ba.png">
##

**Comment by srizzo** [comment]
multipart/form-data works fine

**Comment by bkimminich** [comment]
Reproduced problem with exact request used in bug report:

```
C:\Data\Github\juice-shop\node_modules\sequelize\lib\utils\parameter-validator.js:76
    var _value       = (value === null) ? 'null' : value.toString()
                                                         ^

TypeError: value.toString is not a function
    at validate (C:\Data\Github\juice-shop\node_modules\sequelize\lib\utils\parameter-validator.js:76:58)
    at Object.module.exports.check (C:\Data\Github\juice-shop\node_modules\sequelize\lib\utils\parameter-validator.js:30:10                              )
    at Object.module.exports.validateParameter (C:\Data\Github\juice-shop\node_modules\sequelize\lib\utils.js:580:31)
    at module.exports.DAOFactory.create (C:\Data\Github\juice-shop\node_modules\sequelize\lib\dao-factory.js:688:11)
    at handleResourceCreate (C:\Data\Github\juice-shop\node_modules\sequelize-restful\lib\router.js:324:8)
    at Router.handleRequest (C:\Data\Github\juice-shop\node_modules\sequelize-restful\lib\router.js:67:34)
    at C:\Data\Github\juice-shop\node_modules\sequelize-restful\lib\index.js:10:16
    at C:\Data\Github\juice-shop\node_modules\sequelize-restful\node_modules\connect\lib\middleware\bodyParser.js:56:25
    at IncomingMessage.<anonymous> (C:\Data\Github\juice-shop\node_modules\sequelize-restful\node_modules\connect\lib\middl                              eware\urlencoded.js:73:11)
    at emitNone (events.js:67:13)

npm ERR! Windows_NT 6.1.7601
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "start"
npm ERR! node v4.2.3
npm ERR! npm  v2.14.7
npm ERR! code ELIFECYCLE
npm ERR! juice-shop@1.24.1 start: `node app`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the juice-shop@1.24.1 start script 'node app'.
npm ERR! This is most likely a problem with the juice-shop package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node app
npm ERR! You can get their info via:
npm ERR!     npm owner ls juice-shop
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     C:\Data\Github\juice-shop\npm-debug.log
```

**Comment by bkimminich** [comment]
Seems to have same underlying problem as #131.

**Comment by bkimminich** [comment]
Tried `sequelize-restul-extended` but it causes the same crashes when a non-empty `POST` request is sent via `x-www-form-urlencoded`.

**Comment by bkimminich** [comment]
Fixed in upcoming v6.0.0 release through #167.
